---
int32 servo_id    # サーボID（負の値は失敗の場合）